packages/components/action-select/src/action-select.css

```diff
@@ -5,3 +5,11 @@
 		cursor: pointer;
 	}
 }
+
+.relative {
+	position: relative;
+}
+
+.font-size-0 {
+	font-size: 0;
+}
```

Comment:
Why not join these two classes in one? Both of them are applied to only one element.

Reply:
I deliberately did that, since I wanted to talk to you about atomic styling approach similar to Tailwind. I tried it out in my pet project and started to like it. Seems this approach can be a good fit for Elements. I think we should talk about it in one of our TechTalks.
I'll change this back to single class name for now, for sake of consistency.